fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -37,8 +37,11 @@
 
 enum class RelocateReason { INVALID = -1, OTHER, REBALANCE_DISK, REBALANCE_READ };
 
+enum class RelocateOperation { INVALID = -1, MOVE, MULTIPLY_REPLICAS, REDUCE_REPLICAS };
```

Comment:
I am assuming this is a quick-and-dirty hack to get the dynamic replication code exercised. Eventually, we want to collapse RelocateReason into DataMovementReason. Ideally, this should also go that way.

Reply:
delete `RelocateOperation`, split `REBALANCE_READ` from `RelocateReason` into 
```
MOVE_SHARD,
MULTIPLY_SHARD,
REDUCE_SHARD,
```